src/identity/IdentityPage.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('Identity page', () => {
         <Route path="/users/:userId">
           <IdentityPage />
         </Route>
-      </Router>,
+      </Router>
```

Comment:
These didn't align with the Prettier config of `trailingComma: "es5"`, so got auto-updated.